controller/csv/csv.go

```diff
@@ -0,0 +1,24 @@
+package csv
```

Comment:
For controllers it's more common to see just one package `controller` with all of the handlers. And just group common handlers within different files

Reply:
*